lib/screens/v2/candidate_generator/widgets/train_crowding.ex

```diff
@@ -239,9 +244,13 @@ defmodule Screens.V2.CandidateGenerator.Widgets.TrainCrowding do
 
   defp check_time_based_heuristic(
          cached_prediction,
-         common_params
+         common_params,
+         previous_departure_time_cushion
        ) do
-    show_widget_after_dt = DateTime.add(cached_prediction.departure_time, -10)
+    # cached_prediction.departure_time minus previous_departure_time_cushion is when we expect crowding to be reliable.
+    # When now >= this time, show the widget.
```

Comment:
Still refers to showing the widget. Perhaps instead:

```suggestion
    # When now >= this time, log it.
```